src/udev/udevadm-info.c

```diff
@@ -22,6 +22,25 @@
 #include "udevadm.h"
 #include "udevadm-util.h"
 
+typedef enum action_type {
+        ACTION_QUERY,
+        ACTION_ATTRIBUTE_WALK,
+        ACTION_DEVICE_ID_FILE,
+} ActionType;
```

Comment:
hmm, both a lowercase name and an uppercase name? maybe clean this up and just do `typedef enum ActionType {…} ActionType;`  how we do it usually?